src/child/actions/window.js

```diff
@@ -43,3 +44,21 @@ export function close() {
         type: ACTION_TYPES.CLOSE
     };
 }
+
+export function resizeToCompact() {
+    return dispatch => {
+        const compactWindowDimensions = configService.getCompactWindowDimensions();
+        fin.desktop.Window.getCurrent().resizeTo(compactWindowDimensions[0], compactWindowDimensions[1], 'top-right', () => {
+            dispatch(compact());
```

Comment:
Probably don't need to export `compact()` and `expand()` now that we have `resizeToCompact()` and `resizeToDefault()`?